lua/autorun/server/bounties.lua

```diff
@@ -23,39 +27,57 @@ end
 function AddBounty(ply,target,amount)
 	if !ply:IsPlayer() or !target:IsPlayer() or type(amount) != "number" then return end
 	amount = amount > 0 and amount or 0
-	if ply:PS_HasPoints(amount) then
-		SetPlayerBounty(target,GetPlayerBounty(target) + amount)
+	if BountyLimit[ply:SteamID()] == nil then BountyLimit[ply:SteamID()] = 0 end
+	local total = BountyLimit[ply:SteamID()] + amount
```

Comment:
fyi you can just do like (BountyLimit[ply:SteamID()] or 0) to have a default value